doxygen/mbedtls.doxyfile

```diff
@@ -32,10 +32,13 @@ DOT_TRANSPARENT        = YES
 # but clang -Wdocumentation doesn't (since Clang 15, for \retval).
 #   https://github.com/Mbed-TLS/mbedtls/issues/6960
 #   https://github.com/llvm/llvm-project/issues/60315
-# As a workaround, when documenting the status codes that a function can
-# return, if you don't have anything to say beyond the status code's
-# description, you can write something like
-#    \retval #PSA_ERROR_INSUFFICIENT_MEMORY \emptydescription
+# We often use \retval declarations with just a constant name to
+# document which error codes a function can return. If the documentation
+# of the error code is enough to explain the error, then an empty
+# description on the \retval statement is ok. However, the source code
+# of the description needs to be made non-empty to pacify Clang.
+# In such cases, you can write something like
+#     \retval #PSA_ERROR_INSUFFICIENT_MEMORY \emptydescription
```

Comment:
`s/If the documentation of the error code is enough to explain the error/If the name of the error code is enough to explain the error/`, but also

```suggestion
# We often use \retval declarations with just a constant name to
# document which error codes a function can return. If the name
# of the error code is sufficient to explain the error, then an empty
# description on the \retval statement is ok. However, the source code
# of the description needs to be made non-empty to pacify Clang.
# In these cases, use the \emptydescription alias defined below, e.g.
#     \retval #PSA_ERROR_INSUFFICIENT_MEMORY \emptydescription

```

Reply:
I disagree: the name of the error code is not necessarily sufficient documentation. The documentation of the error code is sufficient documentation.

Reply:
but in this case the documentation is solely the name - that's the point: the description is empty

Reply:
No: the documentation of the return value is a _hyperlinked_ name. `\emptydescription` is meant to be used when the text we would write would be essentially what you see when you follow the hyperlink.

Reply:
Paul has explained it to me: I was interpreting "documentation" as "the documentation right here" as opposed to "the standard documentation for this error code" - which, as you've said, Doxygen will hyperlink you to (which of course doesn't happen if you're reading the source code, which is where I spend 99% of my time).

I don't think my interpretation is unreasonable, and given it needed explanation, I still don't think this is very clear

Reply:
At this point I don't know how to make it clearer. I don't see anything ambiguous about “the documentation of the error code”. Can you please suggest alternative wording (that doesn't change the meaning)?

Reply:
Hmm, my confusion comes from the fact that we're talking about the documentation being written for a function, and referring to the (separate) documentation of the error code. How about "If the standard description of the error code is enough..."? Otherwise I guess just leave it as it is...